java/src/Ice/src/main/java/com/zeroc/Ice/BZip2.java

```diff
@@ -24,7 +24,7 @@ public static Buffer compress(Buffer buf, int headerSize, int compressionLevel)
             //
             data = buf.b.array();
             offset = buf.b.arrayOffset();
-        } catch (java.lang.Exception ex) {
+        } catch (Exception ex) {
```

Comment:
Now that there's no `Exception` class that we define, we no longer need to qualify the built in `Exception` type.